src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -273,4 +283,64 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       postResponse.entity.asString.contains(Conflict.reason)
   }
 
+  def syncWhitelist: Unit = {
+    val (bucket, file) = (FireCloudConfig.Nih.whitelistBucket, FireCloudConfig.Nih.whitelistFile)
+    val whitelist = Source.fromInputStream(HttpGoogleServicesDAO.getBucketObjectAsInputStream(bucket, file)).getLines().toSet
```

Comment:
here, or maybe in HttpGoogleServicesDAO, trap and catch any exceptions in reading the whitelist from the bucket? My concern here is that if we let the raw exception bubble up, we might be exposing sensitive information (bucket name, or even worse some of the secret credentials) in a stack trace visible by the end user.